src/app/shared/components/add-evaluator/add-evaluator.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { User } from '@entity';
+import { RelevancyService } from 'app/core/relevancy.service';
+import { ToastrOvenService } from 'app/shared/modules/toaster/notification.service';
+import { LearningObject } from 'entity/learning-object/learning-object';
+
+@Component({
+  selector: 'clark-add-evaluator',
+  templateUrl: './add-evaluator.component.html',
+  styleUrls: ['./add-evaluator.component.scss']
+})
+export class AddEvaluatorComponent implements OnInit {
+
+  selectedEvaluators: User[];
+
+  @Input() highlightedLearningObjects: LearningObject[];
+  @Output() close: EventEmitter<void> = new EventEmitter();
```

Comment:
You can enforce the type 'boolean' here instead of void

Reply:
It doesn't need to be void, it would be redundant.

Reply:
I agree with Chris.